planning/autoware_path_optimizer/src/mpt_optimizer.cpp

```diff
@@ -1433,8 +1433,6 @@ MPTOptimizer::ConstraintMatrix MPTOptimizer::calcConstraintMatrix(
       lb(A_rows_end + i) = ref_steer_angle - mpt_param_.max_steer_rad;
       ub(A_rows_end + i) = ref_steer_angle + mpt_param_.max_steer_rad;
     }
-
-    A_rows_end += N_u;
```

Comment:
This part is creating constraints for the optimization and `A_rows_end` remembers the size of constraints.
When we add a new constraint in the latter part, the removed line is important to know the size of constraints.

Isn't it possible to keep the current implementation even though it is useless currently.

Reply:
Thanks for the confirmation.

If it is left intentionally, it can be suppressed in the comments and we will change it that way.

Reply:
Thank you.